skimage/transform/_orientation.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+
+
+def _normalize(x):
+    """Normalizes an array.
+
+    Parameters
+    ----------
+    x : array_like
+        Array to normalize.
+
+    Returns
+    -------
+    u : array
+        Unitary array.
+
+    Examples
+    --------
+    >>> x = np.arange(5)
+    >>> uX = _normalize(x)
+    >>> np.isclose(np.linalg.norm(uX), 1)
+    True
+    """
+    u = np.asarray(x)
```

Comment:
Since `u` is the return value, I would call this something else, `arr` or `v`, for example.